### ISSUE TITLE
[Bug] Quickstart Failure

### ISSUE THREAD

**Issue by DavidYKay** [mariadb-operator/mariadb-operator#428]
**Documentation**
- [x] I acknowledge that I have read the relevant [documentation](https://github.com/mariadb-operator/mariadb-operator/tree/main/docs).

**Describe the bug**

I am able to successfully apply the config folder:

`kubectl apply -f examples/manifests/config`

However, I am unable to apply the database config.

These attempts: 

```
kubectl apply -f examples/manifests/mariadb_minimal.yaml 
kubectl apply -f examples/manifests/mariadb.yaml
kubectl apply -f examples/manifests/mariadb_full.yaml
```

all lead to the following error message:

```
error: resource mapping not found for name: "mariadb" namespace: "" from "examples/manifests/mariadb_full.yaml": no matches for kind "MariaDB" in version "k8s.mariadb.com/v1alpha1"
ensure CRDs are installed first
```

**Expected behaviour**

I expect to create a database / StatefulSet in my cluster.

**Steps to reproduce the bug**

1. `helm install mariadb-operator mariadb-operator/mariadb-operator`
2. `kubectl apply -f examples/manifests/mariadb.yaml`

**Debug information**

- Related object events:
```bash
kubectl get events --field-selector involvedObject.name=<mariadb-resource-name>
kubectl get events --field-selector involvedObject.name=<backup-resource-name>
kubectl get events --field-selector involvedObject.name=<restore-resource-name>
```
- `mariadb-operator` logs. Set the `--log-level` to `debug` if needed.

**Environment details**:
- Kubernetes version: v1.28.7+k3s1
- Kubernetes distribution: k3s
- mariadb-operator version: [Version number]
- Install method: helm
- Install flavor: minimal

**Additional context**

FYI my installation is reported as successful:

```
NAME: mariadb-operator
LAST DEPLOYED: Sat Mar  2 17:10:10 2024
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
mariadb-operator has been successfully deployed! 🦭
```

I have also attempted installation by downloading the manifest and invoking `kubectl apply -f`. 

This failed with the following error message:

`The CustomResourceDefinition "mariadbs.mariadb.mmontes.io" is invalid: metadata.annotations: Too long: must have at most 262144 bytes`

**Comment by uozalp** [comment]
The CRD has been renamed, but the released version still uses the old CRD names. It should have been done in a branch and committed when version 0.26 was released. Rename the CRD in the example file to: `mariadb.mariadb.mmontes.io`

**Comment by DavidYKay** [comment]
Fixed by upgrading to 0.0.26 via helm. Thank you! 🙇‍♂️

**Comment by mmontes11** [comment]
Sorry for the inconvenience @DavidYKay and thanks for @uozalp the resolution. Next time we will keep the new examples in a branch before running the release workflow, not doing so seems to disturb a lot of people unfortunately, noted 👍🏻
